### PR TITLE
IBX-777 [CI] Moved CS and PHPUnit jobs to Github Actions

### PR DESCRIPTION
**Pull request by pbisaga**
Adjusting CI test to GitHub Actions instead of Travis
https://issues.ibexa.co/browse/IBX-777

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -70,4 +62,4 @@ after_failure:
   - docker ps -s
 
 after_script:
-  - if [ "${BEHAT_OPTS}" != "" ] ; then bin/ezreport; fi 
\ No newline at end of file
+  - bin/ezreport
```
Comment (mnocon) [inline .travis.yml]: Nitpick: missing newline

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# .github/workflows/ci.yaml
```
Comment (mnocon) [inline .github/workflows/ci.yaml]: Nitpick: we can remove this comment

**Review by mnocon — APPROVED** [review]

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (Steveb-p) [reply .github/workflows/ci.yaml]:
```suggestion
```

File: .travis.yml (hunk 1 above)
Comment (Steveb-p) [reply .travis.yml]:
```suggestion
  - bin/ezreport

```

**Review by Steveb-p — APPROVED** [review]

**Review by bogusez — APPROVED** [review]

File: composer.json [hunk 3]
```diff
@@ -36,7 +36,9 @@
         }
     },
     "scripts": {
-        "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
+        "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating",
```
Comment (Steveb-p) [inline composer.json]:
```suggestion
        "fix-cs": "php-cs-fixer fix -v --show-progress=estimating",
```

**Pull request merged**